src/map/house/house.cpp

```diff
@@ -122,7 +122,9 @@ void House::updateDoorDescription() const {
 	ss << " It is " << houseTiles.size() << " square meters.";
 	const int32_t housePrice = getPrice();
 	if (housePrice != -1) {
-		ss << " It costs " << formatNumber(getPrice()) << " gold coins.";
+		if (owner == 0) {
```

Comment:
Shouldn't the whole thing not show if there is an owner? Maybe if (housePrice != -1 && owner != 0)?